lib/ahoy/intercom/tracker.rb

```diff
@@ -0,0 +1,26 @@
+require 'intercom'
+require 'ahoy/intercom/client'
+
+module Ahoy
+  module Intercom
+    module Tracker
+      include Client
+
+      def track(event_name, properties = {})
```

Comment:
docs of ahoy custom store say that method should be named "track_event" (and "track_visit"). Why you've used 'track' ?